README.md

```diff
@@ -70,12 +72,13 @@ They are not suitable for most production use-cases because they keep all data i
    `ferretdb` runs FerretDB.
 
 2. Start services with `docker compose up -d`.
+   To be sure that you've fetched the latest version you can precede the command with `docker compose pull`.
```

Comment:
```suggestion
   Fetch the latest version with the command `docker compose pull`.
```
Just less words.

Reply:
In that case I believe we should put `docker compose pull` before `docker compose up`, as it needs to be ran first. The other thing is that it's optional to run `docker compose pull`

Reply:
Yeah we can do that :+1: